docs/setup/env-setup.md

```diff
@@ -87,6 +87,24 @@ Without setting the enviromnent minikube might not find the docker images when d
 1. run the image build from the yunikorn-k8shim repository root: `make image`
 1. deploy the scheduler as per the normal instructions.
 
+## Debug code locally
+
+Note, this instruction requires you have GoLand IDE for development.
+
+In GoLand, go to yunikorn-k8shim project. Then click "Run" -> "Debug..." -> "Edit Configuration..." to get the pop-up configuration window,
+
+![Debug Configuration](../images/goland_debug.jpg)
+
+The highlighted fields are the configurations you need to add. These include:
```

Comment:
The option `Go Build` per the left hand side is not open/added by default
might need to explain that